### PR TITLE
Revo: Accept 8MByte flash chips too

### PR DESCRIPTION
**Pull request by tracernz**
They're shipping on some clones. See https://forum.dronin.org/forum/d/273-flip32-omnibus-f4-boards-on-dronin/42

### CODE REVIEW

**Comment by tracernz** [comment]
jenkins, build artifacts please

File: flight/targets/revolution/board-info/board_hw_defs.c [hunk 1]
```diff
@@ -389,22 +389,31 @@ static const struct flashfs_logfs_cfg flashfs_settings_cfg = {
...
- */
-static const struct pios_flash_jedec_cfg flash_n25q128_cfg = {
-	.expect_manufacturer = JEDEC_MANUFACTURER_ANY,
-	.expect_memorytype   = JEDEC_MEMORYTYPE_ANY,
-	.expect_capacity     = 0x18,
-	.sector_erase        = 0xD8,
+static const struct pios_flash_jedec_cfg flash_chip_cfgs[] = {
+	{
+		/* M25P 2MByte chip used in original revo design */
+		.expect_manufacturer = JEDEC_MANUFACTURER_ANY,
```
Comment (tracernz) [inline flight/targets/revolution/board-info/board_hw_defs.c:395]: Oops, I didn't mean to change this... I'm not concerned about it though in any case?

Comment (mlyle) [reply flight/targets/revolution/board-info/board_hw_defs.c:395]: The only risk is we talk to a flash chip and send it commands that do something bad. But doesn't seem like too much of a risk---how many 2mbyte not compatible boards are going to show up?

**Review by mlyle — APPROVED** [review]
If this fixes problem, ok to merge.

**Comment by mlyle** [comment]
Not sure why there is no Jenkins comment--- it looks healthy.

Jenkins, build artifacts please

**Comment by yds** [comment]
Both of my OmnibusF4 FCs now boot just fine! :+1:

- Winbond 25Q128FVSG 1649 - 15728640 Log partition in dRonin
- Winbond 25Q64FVSIG 1705 - 7340032 Log partition in dRonin

Thank you, @tracernz! Hope this makes into Neat.1

**Comment by mlyle** [comment]
Thanks for the test.  Hopefully we don't need a neat.1 but will tag this just in case there is one for other reasons.

**Pull request merged**